src/components/Databoard-review.js

```diff
@@ -0,0 +1,133 @@
+
+  const [homeCornersData, setHomeCornersData] = useState([]);
+  const [awayCornersData, setAwayCornersData] = useState([]);
+
+  useEffect(() => {
+    console.log("Fetching data");
+    fetch(process.env.REACT_APP_API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        let matchData = data.match;
+        let liveData = matchData.liveData;
+        let lineupData = liveData.lineups;
+        let homeData = lineupData.home.stats;
+        let awayData = lineupData.away.stats;
+
+        setHomePossessionData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "possessionPercentage";
+            })[0][timekey]
+          )
+        );
+        setAwayPossessionData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "possessionPercentage";
+            })[0][timekey]
+          )
+        );
+
+        setHomeShotData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "totalScoringAtt";
+            })[0][timekey]
+          )
+        );
+        setAwayShotData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "totalScoringAtt";
+            })[0][timekey]
+          )
+        );
+
+        setHomeShotOnTargetData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "ontargetScoringAtt";
+            })[0][timekey]
+          )
+        );
+        setAwayShotOnTargetData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "ontargetScoringAtt";
+            })[0][timekey]
+          )
+        );
+
+        setHomeCornersData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "wonCorners";
+            })[0][timekey],
+            0
+          )
+        );
+        setAwayCornersData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "wonCorners";
+            })[0][timekey],
+            0
+          )
+        );
+        setDataLoaded(true);
+      });
+  }, [timekey]);
+  return dataLoaded ? (
+    <div className="board-display">
+      <ResultSegment
+        type="Possession"
+        homeMax={homePossessionData + awayPossessionData}
+        homeValue={homePossessionData}
+        awayMax={homePossessionData + awayPossessionData}
+        awayValue={awayPossessionData}
+      ></ResultSegment>
+
+      <ResultSegment
+        type="Shots"
+        homeMax={homeShotData + awayShotData}
+        homeValue={homeShotData}
+        awayMax={homeShotData + awayShotData}
+        awayValue={awayShotData}
+      ></ResultSegment>
+
+      <ResultSegment
+        type="Shots on Target"
+        homeMax={homeShotOnTargetData + awayShotOnTargetData}
+        homeValue={homeShotOnTargetData}
+        awayMax={homeShotOnTargetData + awayShotOnTargetData}
```

Comment:
here your homeMax and awayMax are always the same value, so I don't think there's any point passing both through as separate props. You're passing though the homeValue and awayValue anyway - so I think defining the total in ResultSegment is fine:
`const totalShotsOnTarget = homeValue + awayValue`